### DISCUSSION TITLE
After renaming controls in MainPage.xaml, compiler errors

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#14031] [Q&A] [answered]
I renamed a number of menu items. Now I get a series of errors when building, viz
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0103	The name '_openFileButtonSubject' does not exist in the current context	PelotonIDE (net7.0-windows10.0.19041)	C:\Users\bugma\Source\Repos\3rdParty\PelotonIDE\PelotonIDE\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGenerator\MainPage_7f6f72cbf5927d5ac75c91c97515ab80.cs	974	Active

```
However, when I go looking for that symbol, it's nowhere to be found! `C:\Users\bugma\Source\Repos\3rdParty\PelotonIDE>findstr /m /s "_openFileButtonSubject" *.*` finds nothing.

I've cleaned the solution (and deleted the build-generated bin and obj folders) but still getting the error. Ideas?

_LATER_

The `Uno.UI.SourceGenerators` doesn't exist. I didn't remove it.

_MUCH LATER_

I think it has to do with not being thorough in my renaming
```xml
       <Button x:Name="butOpen" AutomationProperties.Name="Open" ToolTipService.ToolTip="Open"
            RelativePanel.RightOf="newFileButton" RelativePanel.Below="menuBar" Click="FileOpen_Click">
            <Button.Content>
                <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE8E5;"/>
            </Button.Content>
        </Button>
```
`RelativePanel.RightOf="newFileButton"` should be `RelativePanel.RightOf="butNew"`

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! This generally happens when there's a relative panel reference to a non-existent element. We're tracking this with https://github.com/unoplatform/uno/issues/11750.

**Reply by jeromelaban** [reply]
Also, the source generator you're seeing is based on Roslyn generators, and you can find the generated sources under the analyzers node in under your project in the solution explorer.
